mw-webapp/src/logic/wayPage/wayStatistics/statisticBlock/statisticItem/StatisticItem.tsx

```diff
@@ -63,8 +63,8 @@ export const StatisticItem = (props: StatisticItemProps) => {
   return (
     <div className={clsx(styles.statisticItem, styles[props.type ?? StatisticItemType.PRIMARY])}>
       <span className={styles.statisticValue}>
-        {props.convertToHours
-          ? `${Math.round(props.statisticItem.value / MINUTES_IN_HOUR)}h`
+        {props.type === StatisticItemType.SECONDARY || props.convertToHours
```

Comment:
Let's move to const this difficult logic)

Reply:
done